packages/app/package.json

```diff
@@ -77,7 +77,7 @@
       "firebase": "11.4.0",
       "iosTarget": "13.0",
       "macosTarget": "10.15",
-      "tvosTarget": "12.0"
+      "tvosTarget": "13.0"
```

Comment:
FirebaseAuth is currently the highest minimum version for iOS, consulted it here and it was 13 for tvOS as well
https://github.com/firebase/firebase-ios-sdk/blob/36da449b15619d301302faeba639c7893d576220/FirebaseAuth.podspec#L24

Which matches Core, so I chose 13
https://github.com/firebase/firebase-ios-sdk/blob/36da449b15619d301302faeba639c7893d576220/FirebaseCore.podspec#L23